static/style.css

```diff
@@ -266,6 +266,14 @@ footer a:hover {
 	}
 }
 
+#album-tracklist{
+	display: flex;
+	flex-direction: column;
+	padding-left: 10em;
```

Comment:
This padding should be decreased on mobile.

Reply:
I've removed this explicit padding and it looks good from my testing now. I see where it's coming from the container's settings, and from the `@media screen` sections. This PR has made me learn a lot of CSS :laughing: